src/styles/app.scss

```diff
@@ -180,6 +198,10 @@ section {
   }
 }
 
+#learn-begin {
+  padding: 30px 0 0 0;
```

Comment:
This approach is really fragile, and I'm sure it will break eventually. I did use it to prototype the change, yes, but in the actual change we land I'd prefer a more robust approach.

Reply:
I agree, I think it's ok to just remove these lines.

Reply:
I removed them and also reduce the min-height of buttons from 100px to 70px.
This works really well, the result can be viewed at http://46.226.108.172:8002/fr/learn as usual.